docs/features.md

```diff
@@ -2,4 +2,35 @@ Click the below for a walk through of the product functionality:
 
 [![PostHog demo](_media/posthog-demo.png)](https://youtu.be/vGy9gzMAAes-Y?t=35s "PostHog Demo Video")
 
-There is a detailed table of the functionality on the [pricing](/pricing) page.
\ No newline at end of file
+There is a detailed table of the functionality on the [pricing](/pricing) page.
+
+Below is a table of PostHog features and how this compares with the functionality in common alternatives.
+
+
+|  Feature 	|   PostHog	| Amplitude  	|  Mixpanel 	| Heap  	|
+|:-:	|:-:	|:-:	|:-:	|:-:	|
+|   Autocapture	|   ✔	|   	|   	|   ✔	|
+|   API 	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   In-App Events Tracking	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Conversion Tracking	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Dashboard	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Data Visualisation	|  ✔ 	|   ✔	|   ✔	|   	|
+|   Cohort Analysis	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Funnel Analysis	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Push Notifications	|   	|   ✔	|   ✔	|   ✔	|
+|   Retention Tracking	|   	|   ✔	|   ✔	|   ✔	|
+|   Revenue Tracking	|   	|   ✔	|   ✔	|   ✔	|
+|   Uninstall Tracking	|   	|   ✔	|   	|   	|
```

Comment:
I can't find this specific amplitude feature but this sounds like generic tracking, in that all of us can do this?

Reply:
I was refering to this - https://amplitude.com/blog/2016/09/14/app-install-uninstall-metrics 

But I can see a route to do this with PostHog can remove the row

Reply:
Hm yeah I don't think this is an actual feature, just "something you can do with amplitude"